<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CredentialsSelectionPage</class>
 <widget class="QWizardPage" name="CredentialsSelectionPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="title">
   <string>Select Credentials</string>
  </property>
  <property name="subTitle">
   <string>Select the credentials you want to transfer to the partner.</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="0" column="0" colspan="3">
    <widget class="QListView" name="listView">
     <property name="contextMenuPolicy">
      <enum>Qt::ActionsContextMenu</enum>
     </property>
     <property name="editTriggers">
      <set>QAbstractItemView::NoEditTriggers</set>
     </property>
     <property name="tabKeyNavigation">
      <bool>true</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>349</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="2">
    <widget class="QToolButton" name="selectButton">
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonFollowStyle</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QToolButton" name="deselectButton">
     <property name="toolButtonStyle">
      <enum>Qt::ToolButtonFollowStyle</enum>
     </property>
    </widget>
   </item>
  </layout>
  <action name="actionSelect_all">
   <property name="icon">
    <iconset theme="edit-select-all">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Select all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="actionDeselect_all">
   <property name="icon">
    <iconset theme="edit-select-none">
     <normaloff>.</normaloff>.</iconset>
   </property>
   <property name="text">
    <string>Deselect all</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Shift+A</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
